_posts/2018-12-07-fast-starting-clojure-lambdas-using-graalvm.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Fast starting Clojure AWS Lambdas using GraalVM and a Lambda custom runtime  
+author: hjhamala, markkuko
+excerpt: Clojure suffers of a slow start up time which makes using it problematic for APIs running in AWS Lambdas. Compiling Clojure to a native binary using GraalVM and running it in a Lambda custom runtime solves the start up problem. 
+tags:
+- AWS
+- Clojure
+- Serverless 
+- Lambda
+- GraalVM
+---
+
+[**Clojure**](https://clojure.org/) is a dynamic Lisp language which is compiled to JVM bytecode for running as a normal JVM application. Starting a Clojure program is pretty slow compared to many other languages. The start up takes at least one second but depending on a program size this could be almost ten seconds.
```

Comment:
```suggestion
[**Clojure**](https://clojure.org/) is a dynamic Lisp language which is compiled to JVM bytecode for running as a normal JVM application. Starting a Clojure program is pretty slow compared to many other languages. The start-up takes at least one second but depending on program size this could be almost ten seconds.
```